css/style.css

```
@@ -0,0 +1,118 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap");
+
+body {
+  margin: 0;
+  font-family: "Poppins";
+  display: grid;
+  grid-template: 0.15fr 0.85fr / 1fr 1fr 1fr;
+  background: #dc0a2d;
+}
+
+a{text-decoration: none;
+  color: inherit
+}
+
+header {
+  grid-area: 1/1/1/4;
+  padding: 4px;
+}
+
+.title {
+  display: flex;
+  align-content: center;
+  margin: 16px 20px 12px;
+}
+
+.title__image {
+  height: 100%;
+  align-self: center;
+}
+.title__text {
+  margin: 0;
+  margin-left: 16px;
+  color: white;
+  font-size: 24px;
+  font-weight: bold;
+}
+
+.searchbar {
+  margin: 0px 16px 16px 16px;
```

```suggestion
  margin: 0 16px 16px 16px;
```

Faça um teste aí se assim também funciona. Geralmente quando o valor é 0, não colocamos a unidade de medida. Se funcionar bacana, arrume os outros também da mesma forma :)